content/software-security/learning-labs/ll202506.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Chainguard Libraries for Python"
+linktitle: "Chainguard Libraries for Python"
+description: "Learning Lab for June 2025 on Chainguard Libraries for Python and Supply Chain Security"
+type: "article"
+date: 2025-06-22T17:00:00+00:00
+lastmod: 2025-06-22T00:00:00+00:00
+draft: false
+tags: ["Learning Labs", "Chainguard Libraries", "Python"]
+menu:
+  docs:
+    parent: "learning-labs"
+weight: 110
```

Comment:
Does this place it on the top of the list and nav .. it should.. latest should be first..